### PR TITLE
Using shared FormatMessage interop

### PR DESCRIPTION
**Pull request by Maxwe11**
Some projects was using its own definition of FormatMessage and its implementation GetMessage. This commit changes it to use the shared definition from Common folder.
Continuation of #681

### CODE REVIEW

**Comment by Maxwe11** [comment]
> nuget used by another process...

Test again please

**Comment by stephentoub** [comment]
@dotnet-bot test this please

File: src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj [hunk 1]
```diff
@@ -22,20 +22,25 @@
...
-      <Link>Common\Interop\Interop.strerror.cs</Link>
+      <Link>Common\Interop\Unix\Interop.strerror.cs</Link>
     </Compile>
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```
Comment (stephentoub) [inline src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj]: Missing newline at end of file

**Comment by stephentoub** [comment]
LGTM.  Thanks for cleaning this all up (including adding "Unix" to the interop paths where applicable.)

File: src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj (hunk 1 above)
Comment (Maxwe11) [inline src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj]: fixed

**Comment by Maxwe11** [comment]
Thanks @stephentoub,  I've added new lines at the end of files where they weren't

**Comment by stephentoub** [comment]
Thanks, @Maxwe11.

**Comment by ellismg** [comment]
LGTM as well.  Thanks for the cleanups.

@Maxwe11 this doesn't merge cleanly anymore.  Can you rebase on-top of master and force push new history?  I'll merge it then.  Thanks!

**Comment by Maxwe11** [comment]
@ellismg done

**Pull request merged**